Lecture4/webservices/webservices/HttpMethods.java

```diff
@@ -0,0 +1,60 @@
+
+package webservices;
+
+import org.apache.http.client.methods.*;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+
+import java.io.IOException;
+import java.util.Map;
+
+
+public class HttpMethods {
+    private static CloseableHttpClient client;
```

Comment:
I'd like to suggest to make it a non-static and make an ability to pass a HttpClient instance from the outside